### DISCUSSION TITLE
PVC failed to attach to node xxx

### DISCUSSION THREAD

**Discussion by liujiabaoaa** [longhorn/longhorn#6449] [Q&A] [answered]
pod的PVC Unable to mount logs as follows

![1](https://github.com/longhorn/longhorn/assets/26737572/0038e394-936d-4c45-b05e-d57043c27c4c)

longhorn的 log：
time="2023-08-03T03:54:16Z" level=warning msg="pvc-32ad6f99-ad36-4c93-9522-b734cb5d5b21-e-5bac120b: time=\"2023-07-27T04:11:05Z\" level=info msg=\"Shutting down iSCSI target iqn.2019-10.io.longhorn:pvc-32ad6f99-ad36-4c93-9522-b734cb5d5b21\""
time="2023-08-03T03:54:16Z" level=warning msg="pvc-32ad6f99-ad36-4c93-9522-b734cb5d5b21-e-5bac120b: time=\"2023-07-27T04:11:05Z\" level=warning msg=\"failed to unbind initiator target id 3\" error=\"failed to execute: tgtadm [--lld iscsi --op unbind --mode target --tid 3 -I ALL], output , stderr tgtadm: this access control rule does not exist\\n: exit status 22\""
time="2023-08-03T03:54:16Z" level=warning msg="pvc-32ad6f99-ad36-4c93-9522-b734cb5d5b21-e-5bac120b: time=\"2023-07-27T04:11:05Z\" level=error msg=\"Failed to init frontend\" error=\"device pvc-32ad6f99-ad36-4c93-9522-b734cb5d5b21: failed to delete target iqn.2019-10.io.longhorn:pvc-32ad6f99-ad36-4c93-9522-b734cb5d5b21: failed to execute: tgtadm [--lld iscsi --op delete --mode logicalunit --tid 3 --lun 1], output , stderr tgtadm: this logical unit is still active\\n: exit status 22\""
time="2023-08-03T03:54:16Z" level=warning msg="pvc-32ad6f99-ad36-4c93-9522-b734cb5d5b21-e-5bac120b: time=\"2023-07-27T04:11:05Z\" level=error msg=\"failed to init frontend: device pvc-32ad6f99-ad36-4c93-9522-b734cb5d5b21: failed to delete target iqn.2019-10.io.longhorn:pvc-32ad6f99-ad36-4c93-9522-b734cb5d5b21: failed to execute: tgtadm [--lld iscsi --op delete --mode logicalunit --tid 3 --lun 1], output , stderr tgtadm: this logical unit is still active\\n: exit status 22\""

May I ask what caused it?

**Comment by c3y1huang** [comment, marked as answer]
[A possible cause and its corresponding explanation](https://github.com/longhorn/longhorn/issues/4959#issuecomment-1331362213).

What is your Longhorn version?

**Reply by liujiabaoaa** [reply]
I  have resolved  Thank you!

**Reply by bluzioo** [reply]
Could you please explain how to fix it？
